MoviesTips/Views/MoviesList.xaml.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using MoviesTips.ViewModels;
+using Xamarin.Forms;
+
+namespace MoviesTips.Views
+{
+    public partial class MoviesList : ContentPage
+    {
+
+        private MoviesViewModel _viewModel = new MoviesViewModel();
+
+
+        public MoviesList()
+        {
+            InitializeComponent();
+
+            // Binds with ModelView class.
+            this.BindingContext = this._viewModel;
+
+            // Show movie info view when list view item is tapped.
+            this.MoviesListView.ItemTapped += async (sender, e) =>
+            {
+                var Result = e.Item as Models.Result;
+
+                // Call API to get this movie information
+                var Movie = await TMDBAPI.API.GetMovieInfoAsync(Result.Id);
```

Comment:
As the API call is made before pushing the screen, there is a little delay when the user presses the list item.